src/os/bluestore/BitMapAllocator.h

```diff
@@ -42,6 +42,7 @@ class BitMapAllocator : public Allocator {
     uint64_t want_size, uint64_t alloc_unit, int64_t hint,
     uint64_t *offset, uint32_t *length);
 
+  using Allocator::alloc_extents;
```

Comment:
yes, it does not need to be virtual. hence removing virtual. Having it makes sense as it eliminates need of one argument.